kolibri/core/device/management/commands/provisiondevice.py

```diff
@@ -15,6 +17,45 @@
 logger = logging.getLogger(__name__)
 
 
+def _check_setting(name, available, msg):
+    if name not in available:
```

Comment:
Does this check if the value is always a boolean (I think all of the settings are only Booleans)

Reply:
The `landing_page` setting also takes an enumerated list of strings. At this point I intentionally omitted value validation because I didn't want to duplicate logic with internal model validation.

There's also even more complex logic because of inter-relationships between settings. Opened this related issue: https://github.com/learningequality/kolibri/issues/7281